app/services/state_file/state_information_service.rb

```diff
@@ -42,6 +50,27 @@ def state_code_from_intake_class(klass)
         end
         state_code.to_s
       end
+
+      def intake_class_from_state_code(state_code)
+        STATES_INFO[state_code][:intake_class]
+      end
+
+      def navigation_from_state_code(state_code)
+        STATES_INFO[state_code][:navigation]
+      end
```

Comment:
this can be added to the dynamically defined methods too i think, just add `:navigation` to the list

Reply:
After discussing with Martha, we are moving navigation out of this class